samples/drumthumper/build.gradle

```diff
@@ -22,6 +22,20 @@ android {
         }
     }
 
+    // This sample will also be published on the Google Play Store which needs a different
+    // application id from "com.google" as this is reserved for official Google products.
+    flavorDimensions "publishing-target"
+    productFlavors {
+        codesample {
```

Comment:
I interpret the doc differently. I think the flavors are for multiple products in the store, like "free" and "pro".
I think all we need to do is set the applicationId in build.gradle to "com.mobileer.drumthumper", which it is currently.

Reply:
By jove, I think you're right! Have changed the package names but left the applicationId as it was. This keeps things very simple - thanks for investigating.